### PR TITLE
Add ``improve_installation`` as a project

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: data/projects.js [hunk 1]
```diff
@@ -306,5 +306,26 @@ coala should support generating metrics for your code.",
...
+		"difficulty" : "medium",
+		"issues" : [
+			"https://github.com/coala/coala-bears/issues/845",
+			"https://github.com/coala/coala-bears/issues/664"
+		],
+		"mentors" : ["Adrianzatreanu", "Udayan12167"],
+		"initiatives" : ["GSoC", "RGSoC"],
+		"tags" : ["Bears", "cib"],
+		"markdown": "improve_installation.md",
+		"collaborating_projects" : ["coala", "cib"]
```
Comment (sils) [inline data/projects.js]: is cib really an own project? It's still coala or do you intend to make cib an own GSoC org in the future? :P

File: data/projects.js [hunk 2]
```diff
@@ -306,5 +306,26 @@ coala should support generating metrics for your code.",
...
 		"collaborating_projects" : ["coala", "GitMate", "IGitt"]
+	},
+	{
+		"name" : "Improve installation of coala and coala-bears",
+		"desc" : "This project is about improving the \
+		[bear installation tool](https://gitlab.com/coala/cib) and \
+		working around platform dependent requirements.",
+		"requirements" : [
+				"The participant should have one patch to [coala-bears](https://github.com/coala/coala-bears) accepted.",
+				"Huge plus is merging [this issue](https://github.com/coala/coala-bears/issues/845)",
```
Comment (sils) [inline data/projects.js]: I'd remove that, sends the message that only that one student who is merging that issue has a good chance of getting this which isn't true

File: data/projects/improve_installation.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+As a start for this project, EACH bear must have all its dependencies solved
+out (>80% do), including the platform dependent ones. For this to be done,
+DistributionRequirement should be reworked.
+
+#### Milestones
+
+##### GSOC 2017 CODING STARTS
+
+* A cEP for cib should already be merged.
+* Merging https://github.com/coala/coala-bears/issues/845
```
Comment (sils) [inline data/projects/improve_installation.md]: you don't merge issues btw., you solve them

File: data/projects/improve_installation.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+##### GSOC 2017 MIDTERM
+
+* Platform dependencies should be working.
+* All bears should have dependencies sorted out.
+* Tests should be written for all the dependencies and the classes.
```
Comment (sils) [inline data/projects/improve_installation.md]: I'd put special emphasis on documenting all requirements classes here and documenting cib very very properly for the final milestone

File: data/projects.js (hunk 1 above)
Comment (Adrianzatreanu) [inline data/projects.js]: Cib is mine!!!!

File: data/projects.js (hunk 2 above)
Comment (Adrianzatreanu) [inline data/projects.js]: Fair enough

**Comment by sils** [comment]
ok we got a conflict, because all projects are in one file :/

**Comment by Adrianzatreanu** [comment]
conflicts fixed, couild you retake a look @sils :) ? cheers

File: data/projects.js [hunk 5]
```diff
@@ -322,5 +322,26 @@ coala should support generating metrics for your code.",
...
+				"The participant should have one patch to [coala-bears](https://github.com/coala/coala-bears) accepted.",
+				"Huge plus is being familiar with the current design of cib",
+				"Knowledge of conda is a bonus, but not required."
+		],
+		"difficulty" : "medium",
+		"issues" : [
+			"https://github.com/coala/coala-bears/issues/845",
+			"https://github.com/coala/coala-bears/issues/664"
+		],
+		"mentors" : ["Adrianzatreanu", "Udayan12167"],
```
Comment (sims1253) [inline data/projects.js]: maybe @fneu I remember you waiting for cib stuff he was doing

File: data/projects/improve_installation.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+fixing bears which have platform requirements (we could use conda).
+
+Currently, cib does basic operations but rudimentary, by parsing strings
+and running "pip install" on packages.
+
+Once this project is done, cib should be able to perform the following:
+
+ - install any number of bears, specified as command line arguments
+ - update to the latest version any number of bears
+ - check if dependencies are fulfilled for any number of bears
```
Comment (sims1253) [inline data/projects/improve_installation.md]: was the "install dependencies" part droped?

File: data/projects/improve_installation.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+As a start for this project, EACH bear must have all its dependencies solved
+out (>80% do), including the platform dependent ones. For this to be done,
+DistributionRequirement should be reworked.
+
+#### Milestones
+
+##### GSOC 2017 CODING STARTS
+
+* A cEP for cib should already be merged.
```
Comment (sims1253) [inline data/projects/improve_installation.md]: Can you change the formulations to eg: "A cEP that describes cib is merged" for all the others also? 
Should sounds like it is not a hard requirement but a (semi-)soft one.

File: data/projects/improve_installation.md (hunk 6 above)
Comment (Adrianzatreanu) [inline data/projects/improve_installation.md]: no it wsant, good that you reminded

**Comment by Adrianzatreanu** [comment]
fixed

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/118/commits/8a2dd978b28d986932e6d74e13f4449909dfbca0

**Comment by sils** [comment]
@Adrianzatreanu rebasing for you

**Comment by sils** [comment]
otherwise we'll get conflicts with my PR

**Pull request closed without merging**